packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1366,6 +1368,50 @@ module.exports = function (log, config, bounces) {
     });
   };
 
+  Mailer.prototype.linkedGoogleAccountEmail = function (message) {
+    log.trace('mailer.linkedGoogleAccountEmail', {
+      email: message.email,
+      uid: message.uid,
+    });
+
+    const templateName = 'linkedGoogleAccount';
+    const links = this._generateSettingLinks(message, templateName);
+    const subject = gettext('Your Firefox account is linked to a Google account');
+    const action = gettext('Manage account');
+    const [time, date] = this._constructLocalTimeString(
+      message.timeZone,
+      message.acceptLanguage,
+      message.date,
+      message.time
+    );
+
+    const headers = {
+      'X-Link': links.passwordChangeLink,
```

Comment:
We can specify `X-Provider-Id` here to let us know what account was linked.

Reply:
Or maybe `X-Linked-Account-Provider-Id` to be more detailed 😅

Reply:
I need help on this part.... would the provider not also be added as a property within `message` like other information?

Reply:
Yes it would, which actually brings up an important note. We would need to add some code [here](https://github.com/mozilla/fxa/blob/97686e4151285eb1ef60b088f1d1cbd7db2d40c8/packages/fxa-auth-server/lib/routes/linked-accounts.ts#L94) to actually send the mail. This email should only be sent for exisiting FxA users adding a new Google account.

Reply:
For `X-Linked-Account-Provider-Id` I think we can use either the id value or the name value. This value will only be exposed in metrics and we can do any transformations there too.